pkg/controller/service/testdata/ingress/basic/expected.golden

```diff
@@ -70,14 +70,14 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   annotations:
-    acorn.io/targets: '{"ci1.acorn.not":{"port":81,"service":"oneimage"},"localhost":{"port":81,"service":"oneimage"},"oneimage-app-name-a5b0aade.local.oss-acorn.io":{"port":81,"service":"oneimage"}}'
+    acorn.io/targets: '{"oneimage-app-name-a5b0aade.local.oss-acorn.io":{"port":81,"service":"oneimage"}}'
```

Comment:
The change is expected. This PR breaks down ingress into custom domain and cluster domain so that we don't apply cert-manager annotation to existing cluster domain that is using certs issued by acorn's letsencrypt integration